apps/pdda/dev/base/kustomization.yaml

```diff
@@ -3,4 +3,8 @@ kind: Kustomization
 resources:
   - ../../base
   - ../../../rbac/dev-role.yaml
+  - ../../../base/workload-identity
 namespace: pdda
+patches:
+  - path: ../../serviceaccount/stg.yaml
+  - path: ../../serviceaccount/stg.yaml
```

Comment:
is that duplication?

Reply:
Looks like it. I ran it a second time because it said I wasn't logged in.  Looks like it added it a second time. I'll remove that now.